templates/client-daemonset.yaml

```diff
@@ -71,6 +71,14 @@ spec:
         - name: config
           configMap:
             name: {{ template "consul.fullname" . }}-client-config
+        {{- if .Values.global.tls.enabled }}
+        - name: tls-ca-cert
+          secret:
+            secretName: {{ template "consul.fullname" . }}-ca-cert
+        - name: tls-client-cert
+          emptyDir:
+            medium: "Memory"
```

Comment:
Using `tmpfs` to store client certs.

Reply:
fancy! Do you know which versions of kube that's supported with?

Reply:
good question! It's supported in 1.13+, don't know about earlier versions.